sdk/cosmos/azure-cosmos/test/test_crud.py

```diff
@@ -1347,10 +1347,12 @@ def __SetupEntities(client):
             return entities
 
         # Client without any authorization will fail.
-        client = cosmos_client.CosmosClient(CRUDTests.host, {}, "Session", connection_policy=CRUDTests.connectionPolicy)
-        self.__AssertHTTPFailureWithStatus(StatusCodes.UNAUTHORIZED,
-                                           list,
-                                           client.list_databases())
+        try:
+            cosmos_client.CosmosClient(CRUDTests.host, {}, "Session", connection_policy=CRUDTests.connectionPolicy)
+            raise Exception("Test did not fail as expected.")
+        except exceptions.CosmosHttpResponseError as error:
+            assert error.status_code == StatusCodes.UNAUTHORIZED
```

Comment:
You should be able to write this as:
```python
with pytest.assertraises(CosmosHttpResponseError) as error:
    CosmosClient(...)
assert error.value.status_code == StatusCodes.UNAUTHORIZED
```

Reply:
changed this to use our test framework's assertions instead now